.github/workflows/branch.yml

```
@@ -32,7 +24,28 @@ jobs:
         run: |
           echo "127.0.0.1 dashboard.openwisp.org api.openwisp.org" | sudo tee -a /etc/hosts
 
-      - name: Build & Publish
-        run: make publish TAG=edge || (docker-compose logs && exit 1)
+      # echo "$DOCKER_HUB_TOKEN" | docker login --username "$DOCKER_HUB_USER" --password-stdin
```

Do we need to keep this line here or can we remove it?

---

It shows the command line equivalent of the below action.

---

```suggestion
      # the following action is equivalent to:
      # echo "$DOCKER_HUB_TOKEN" | docker login --username "$DOCKER_HUB_USER" --password-stdin
```

Let's make it explicit, what do you think?